DfE.FindInformationAcademiesTrusts/Pages/accessibility.cshtml

```diff
@@ -0,0 +1,116 @@
+@page
+
+@{
+  Layout = "_ContentLayout";
+  ViewData["Title"] = "Accessibility statement";
+}
+
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-three-quarters">
+    <h1 class="govuk-heading-l">
+      Accessibility statement
+    </h1>
+    <p class="govuk-body">
+      This accessibility statement applies to content published on Find information about academies and trusts.
+    </p>
+    <p class="govuk-body">
+      This website is run by Department for Education. We want as many people as possible to be able to use this website. For example, that means you should be able to:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>change colours, contrast levels and fonts</li>
+      <li>zoom in up to 300% without the text spilling off the screen</li>
+      <li>navigate most of the website using just a keyboard</li>
+      <li>navigate most of the website using speech recognition software</li>
+      <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+    <p class="govuk-body">
+      We’ve also made the website text as simple as possible to understand.
+    </p>
+    <p class="govuk-body">
+      <a href="https://mcmw.abilitynet.org.uk" class="govuk-link" rel="noopener" target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.
+    </p>
+    <h2 class="govuk-heading-m">
+      How accessible this website is
+    </h2>
+    <p class="govuk-body">
+      We know some parts of this website are not fully accessible:
+    </p>
+    <p class="govuk-body">
+      This section will be updated after the accessibility audit has taken place.
+    </p>
+    <h2 class="govuk-heading-m">
+      Give feedback and contact us
+    </h2>
+    <p class="govuk-body">
+      If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille email <a href="mailto:regionalservices.rg@education.gov.uk." class="govuk-link" rel="noopener" target="_blank">regionalservices.rg@education.gov.uk.</a>
+    </p>
+    <p class="govuk-body">
+      We’ll consider your request and get back to you in 5 working days.
+    </p>
+    <h2 class="govuk-heading-m">
+      Reporting accessibility problems with this website
+    </h2>
+    <p class="govuk-body">
+      We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact:
+      <a href="mailto:regionalservices.rg@education.gov.uk." class="govuk-link" rel="noopener" target="_blank">regionalservices.rg@education.gov.uk.</a>
```

Comment:
Another errant full stop in the link
```suggestion
      <a href="mailto:regionalservices.rg@education.gov.uk" class="govuk-link" rel="noopener" target="_blank">regionalservices.rg@education.gov.uk</a>.
```

Reply:
i moved the full stop out of the link as it looked weird on the page

Reply:
sorted